### ISSUE TITLE
Questions about detection pretrained weights

### ISSUE THREAD

**Issue by Coldfire93** [Jia-Research-Lab/ReviewKD#7]
I want to make sure that the file _mv2-r50.pth_ in the [ detection pretrained weights](https://github.com/dvlab-research/ReviewKD/releases) you provided contains both teacher's and student's weights.

Thank you!

**Comment by akuxcw** [comment]
Yes, I just checked the mv2-r50.pth contains both teacher and student.

**Comment by Coldfire93** [comment]
Hi @akuxcw ,
The size of the mv2-r50.pth is 266MB. The teacher model is about 160MB ( I'm not sure. I used torchvision's pretrained weights before. It is about 160MB. Your teacher's weights come from Detectron2's pretrained detector. Was there a particular reason why you chose to use Detectron2 instead of torchvision?).    So, the student model's size is about 100MB? Is that right? 
Thank you!

**Comment by akuxcw** [comment]
We use the teacher model from Detectron2 because it provides pretrained detectors and we train our student based on Detectron2. I think there are no pretrained detectors in torchvision, torchvision only provides pretrained weights on ImageNet. Which model do you use before? And we use [this model](https://dl.fbaipublicfiles.com/detectron2/COCO-Detection/faster_rcnn_R_101_FPN_3x/137851257/model_final_f6e8b1.pkl) as teacher, which is about 230M, so student's model is about 30M.

**Comment by Coldfire93** [comment]
Hi @akuxcw ,
We used [this model](https://download.pytorch.org/models/fasterrcnn_resnet50_fpn_coco-258fb6c6.pth) as teacher. It provides detectors too.

>  And we use [this model](https://dl.fbaipublicfiles.com/detectron2/COCO-Detection/faster_rcnn_R_101_FPN_3x/137851257/model_final_f6e8b1.pkl) as teacher, which is about 230M

Is the backbone of the link you gave above ResNet101? Could you please send me the link of model with ResNet50 as backbone? Because the mv2-r50.pth - teacher model's backbone is ResNet50.

Thank you!

**Comment by akuxcw** [comment]
Sorry, faster-rcnn-50 is available [here](https://dl.fbaipublicfiles.com/detectron2/COCO-Detection/faster_rcnn_R_50_FPN_3x/137849458/model_final_280758.pkl). And you are right about the model size.
I really don't know torchvision provides pretrained detectors. Thank you for telling me that! And we use detectron2's pretrained weights just because we implement our method based on detectron2.

**Comment by Coldfire93** [comment]
Hi @akuxcw ,
I see. Thank you~

**Comment by twmht** [comment]
@akuxcw 

Did you also switch teacher to training mode for BN when training detector?
